test/state/state.cpp

```diff
@@ -14,6 +14,10 @@ namespace evmone::state
 {
 namespace
 {
+/// Values specified by EIP-4844
+constexpr auto min_blob_gasprice = 1;
```

Comment:
Bring the constants to the single place they are use, which is `if (tx.type == Transaction::Type::blob)`.
Use naming `MIN_BLOB_GASPRICE` or `MinBlobGasPrice`.
They will be still searchable but no need to extend their scope unnecessarily.